.gitmodules

```diff
@@ -7,6 +7,6 @@
 [submodule "dev/speculos"]
 	path = dev/speculos
 	url = https://github.com/LedgerHQ/speculos
-[submodule "dev/blue-loader-python"]
-	path = dev/blue-loader-python
-	url = https://github.com/LedgerHQ/blue-loader-python
+[submodule "dev/sdk/nanosplus-secure-sdk"]
```

Comment:
The reason for it is a bit complicated :thinking: 

I experienced that it can be problem to use a different SDK for compilation than for speculos.

For instance, the SDK inside the builder uses 1.0.2 as SDK for the nanosplus, but the cxlib in `speculos:latest` is 1.0.

That's why the simulator doesn't work (only black screen).

And for this reason, the shell-script tries to get is consistent as possible by using submodules for the SDK and use the same SDK version for speculos.

I would prefer to get rid of the complicated shell code, though ... but it seems nanosplus-apps compiled with 1.0.2 SDK don't run with 1.0 cxlib in Speculos :see_no_evil:

Reply:
submodules were removed in  https://github.com/iotaledger/ledger-iota-app/pull/34